drivers/led/lp3943.c

```diff
@@ -26,6 +26,12 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(lp3943);
 
+#ifdef CONFIG_HAS_DTS_I2C
+#define CONFIG_LP3943_DEV_NAME			DT_TI_LP3943_0_LABEL
```

Comment:
Hardcoding the instance in driver doesn't seem to be a good idea! What if two LP3943 devices are connected to two different I2C buses? One of them will be instance 1. The selection of devices should stay at the board level.

Reply:
I agree, but the driver it self is limited to a single instance.  If a system had 2 LP3943 the driver should be extended to support 2 instances.